dashboard/app/models/pd/application/teacher1819_application.rb

```diff
@@ -400,5 +402,22 @@ def principal_approval_url
     def check_idempotency
       Pd::Application::Teacher1819Application.find_by(user: user)
     end
+
+    def meets_criteria
+      responses = Teacher1819ApplicationConstants::CRITERIA_SCORES.merge(response_score_hash).values
+      if responses.uniq == [YES]
+        # If all resolve to Yes, applicant meets criteria
+        true
+      elsif responses.include? NO
+        # If any are No, applicant does not meet criteria
+        false
+      else
+        nil
+      end
+    end
```

Comment:
Should this be `meets_criteria?`?

Why would we want it to return nil, rather than false? Is it important to distinguish these 2 states?
If we are to have 3 states, please add a function header with a `@return` explanation.

Reply:
Upon further consideration, might as well return an enum to include "not determined" as a response